inc/plugins/MybbStuff/MyAlerts/templates/page.html

```diff
@@ -27,6 +27,8 @@
                         <strong>{$lang->myalerts_page_title}</strong>
 
                         <div class="float_right">
+                            <a href="{$mybb->settings['bburl']}/alerts.php?action=delete_read&amp;my_post_key={$mybb->post_code}" onclick="return confirm('{$lang->myalerts_delete_read_confirm}')">{$lang->myalerts_usercp_nav_delete_read}</a> | 
```

Comment:
I'm not sure if it's worth adding separate language variables for this or renaming the ones used - these links aren't in the "user cp nav".

Reply:
I will add a new one, give me few minutes